apps/frontend/src/app/5_pages/RewardsPage/components/StabilityPool/StabilityPool.tsx

```diff
@@ -114,6 +113,80 @@ export const StabilityPool: FC = () => {
     ],
   );
 
+  const rows = useMemo(() => {
+    if (amount.isZero() && decimalic(sovGain).isZero()) {
+      return [];
+    }
+
+    return [
+      {
+        type: t(translations.rewardPage.stabilityPool.stabilityPoolRewards),
+        amount: (
+          <AmountRenderer
+            value={amount}
+            suffix={BITCOIN}
+            precision={BTC_RENDER_PRECISION}
+            dataAttribute="rewards-amount"
+          />
+        ),
+        action: (
+          <div className="flex flex-row gap-3">
+            <Button
+              type={ButtonType.button}
+              style={ButtonStyle.secondary}
+              text={t(translations.rewardPage.stabilityPool.actions.withdraw)}
+              onClick={handleWithdraw}
+              disabled={claimDisabled}
+              dataAttribute="rewards-withdraw"
+            />
+            {isOpenTroveExists && Number(amount) > 0 && signer && (
```

Comment:
```suggestion
            {isOpenTroveExists && !amount.isZero() && signer && (
```
`amount` is `Decimal` so casting it to `Number` is unnecessary.